ERCS/erc-7786.md

```diff
@@ -115,13 +115,16 @@ A **Destination Gateway** is a contract that implements a protocol to validate m
 
 The protocol MUST ensure delivery of a sent message to its **receiver** using the `IERC7786Receiver` interface (specified below), which the receiver MUST implement.
 
-Once the message can be safely delivered (see Properties), the gateway MUST invoke `executeMessage` with the message contents.
+Once the message can be safely delivered (see Properties), the gateway MUST invoke `executeMessage` with the message identifier and contents, UNLESS not invoking this function was explicitly requested by the sender (through some attribute) or the receiver.
```

Comment:
```suggestion
Once the message can be safely delivered (see Properties), the gateway MUST invoke `executeMessage` with the message identifier and contents, unless the sender or the receiver explicitly requested otherwise.
```